crawler/filter.go

```diff
@@ -0,0 +1,33 @@
+package crawler
+
+import (
+	"net/url"
+)
+
+type NoneFilter struct{}
+
+func (filter NoneFilter) Filter(urls []string) []string {
+	return urls
+}
+
+type CrossDomainFilter struct {
```

Comment:
Should we move types into a specific location and function into other file

Reply:
discussed.